packages/web-components/fast-components/src/data-grid/data-grid.stories.ts

```diff
@@ -215,7 +215,7 @@ const buttonCellTemplate = html<DataGridCell>`
                 x.columnDefinition === null ||
                 x.columnDefinition.columnDataKey === null
                     ? null
-                    : x.rowData[x.columnDefinition.columnDataKey]}
+                    : x.rowData[x.columnDefinition.columnDataKey as keyof object]}
```

Comment:
This should be a safer type casting:
```suggestion
                    : x.rowData[
                          x.columnDefinition.columnDataKey as keyof typeof x.rowData
                      ]}
```

It looks like TS is treating `object` as `{}`, as evidenced by the emitted error:

```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{}'.
  No index signature with a parameter of type 'string' was found on type '{}'.ts(7053)
```

@chrisdholt perhaps we could set `"suppressImplicitAnyIndexErrors": true` which would disable these errors in the tsconfig? Casting as `object` or `keyof typeof x.rowData` seems a little excessive.

Reply:
I agree with above ^ Having to do the above seems a tad aggressive 😅  I'll wait for the discussion to be had before tackling this specific change